tests/ui/cognitive_complexity.rs

```diff
@@ -133,16 +133,6 @@ fn bloo() {
     }
 }
 
-#[clippy::cognitive_complexity = "0"]
-fn lots_of_short_circuits() -> bool {
-    true && false && true && false && true && false && true
```

Comment:
keep these tests

Reply:
Why? Currently there's no criteria in the draft for this pattern. 
Also (if I understood the point of those tests correctly) checking for short-circuits would require value information, would it not?

Reply:
Just so we know this has a CoC of 1

Reply:
Done :D

Reply:
No wait, I've implemented it, but that test in particular I didn't

Reply:
Okay, now it's done ^^